packages/openactive-broker-microservice/app.js

```diff
@@ -681,12 +673,16 @@ app.get('/dataset-site', function (req, res) {
   res.json(datasetSiteJson);
 });
 
-function mapToObject(map) {
+/**
+ * @param {Map | Set} map
+ * @returns {{[k: string]: any} | number}
+ */
+function mapToObjectSummary(map) {
```

Comment:
Because it doesn't strictly just convert a Map to an object - it also replaces any Sets with their size, I was more into this name